### DISCUSSION TITLE
Switch from Raspi4 to Raspi5

### DISCUSSION THREAD

**Discussion by quarreller** [raspiblitz/raspiblitz#4511] [Q&A]
Currently I have a Raspi4 since 2 days with the new Raspiblitz version raspiblitz-fat-v1.11.0-2024-03-28.img.gz

Everything works with lightning and co and now I want to switch to a Raspi5 8GB.

What do I have to do?

Can I simply insert the SD card into the Raspi5?
plug the ssd hard drive back in and start the Raspi5?

Would that work?

Thanks for your help and answer

**Comment by rootzoll** [comment]
raspiblitz-fat-v1.11.0-2024-03-28.img.gz was not official yet (was uploaded for internal testing) ... hopefully final release will be up in a matter of days.

But then you can just power off, switch the rp4 for the rp5 (USB-SSD stays the same - make sure to use the new stronger power supply) an the you start up with a fresh (final) v1.11 sd card image.

**Comment by quarreller** [comment]
Hello,
did not work for me

I shutdown my raspi4 over ssh
Flashed a new sd card with 1.11.0  and put it in my  rp5
Pluged in the usb ssd and lan cable and powered on

The problems are
1.The recommaded lcd(works on rp4)stays always black
2.Over ssh connect it told me insert an hdd/ssd

So i shutdown again and plug out and in the ssd and powered on again

Problems are
1.The recommaded lcd(works on rp4)stays always black
2. Over ssh connect i had now  the options to set up a new raspiblitz or import  a migration file

So i end here.....and bring on my rp4 with the new flashed sd card
1.Lcd comes up
2.Update  was as suspectet....set a new passwort A and etc was recovered...
and raspiblitz works as it shut.

So why did not work my switch from 4 to 5?
Thank you for answering

**Comment by quarreller** [comment]
Hello,
today I tried again with another sd card, the old ssd and the rpi5.
What can I say.
Lcd works and ssd was recognised.
Update and recovery started and in the end raspiblitz v.1.11.0 works as it should with the rpi5.
No idea what the problem was.
But in the end everything works and I'm happy.

**Reply by norskno** [reply]
Can you see any performance diffrence? :)
